src/Input.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function Input({input,onAdd, onChange}) {
+
+  return (
+    <div>
+
+
+    <input type='text'
+    placeholder='할 일을 입력해 주세요.'
+    value = {input}
+    onChange={onChange}
+    >
```

Comment:
```suggestion
    <input 
    type='text'
    placeholder='할 일을 입력해 주세요.'
    value = {input}
    onChange={onChange}
    >
```

`props` 가 많아서 줄 바꿈을 해줄 때는 위와같이 formatting 해주는게 코드 가독성이 더 좋을 것 같아요 :)